flake.lock

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Usually lock files are not committed for libraries. Not sure if this should be here or not, though?

Reply:
This is lockfile for the CI, not the library. I find it more reliable to have CI state locked and it will also allow you to use the exact same version of dependencies locally in `nix-shell`.